libs/components/src/lib/tag/tag.locale.ts

```diff
@@ -0,0 +1,3 @@
+export interface TagLocale {
+	remove: string;
+}
```

Comment:
Since we're no longer focusing the whole Tag, my suggestion would be to use function here to return localized label and announce it as "Remove Removable tag 1" etc. We're using this approach in a few other localized components, like `SearchableSelect`
```suggestion
export interface TagLocale {
	remove: /* istanbul ignore next */ (label: string) => string;
}
```

Reply:
Btw. In other locale files, we just use 'locale.ts' filename

Reply:
Okay, thanks!